source/beamline/SGM/SGMSampleChamberVacuum.h

```diff
@@ -0,0 +1,49 @@
+#ifndef SGMSAMPLECHAMBERVACUUM_H
+#define SGMSAMPLECHAMBERVACUUM_H
+
+#include "beamline/AMSingleEnumeratedControl.h"
+
+#define SGMSAMPLECHAMBERVACUUM_VENTED_MIN 760.0
+#define SGMSAMPLECHAMBERVACUUM_VENTED_MAX 1000.0 // Has to be larger than any value we would encounter.
+#define SGMSAMPLECHAMBERVACUUM_ROUGHVACUUM_MIN 0.001
+#define SGMSAMPLECHAMBERVACUUM_ROUGHVACUUM_MAX 10.0
+#define SGMSAMPLECHAMBERVACUUM_HIGHVACUUM_MIN 0.000000001 // Has to be smaller than any value we would encounter.
+#define SGMSAMPLECHAMBERVACUUM_HIGHVACUUM_MAX 0.001
```

Comment:
My sense is that the numbers are reasonable. I would say that "high vacuum" and "rough vacuum" are adjacent ranges ... but "vented" is fairly far from "rough vaccum". I'm not really sure what's supposed to be in the middle but it's not a state you want to stay in, that's for sure.